sources/packages/api/src/sfas-integration/sfas-files/sfas-application-record.ts

```diff
@@ -12,91 +12,109 @@ export class SFASApplicationRecord extends SFASRecordIdentification {
    * The unique key/number used in SFAS to identify this individual (individual.individual_idx).
    */
   get individualId(): number {
-    return +this.line.substr(3, 10);
+    return +this.line.substring(3, 3 + 10);
   }
   /**
    * The unique key/number used in SFAS to identify this application (application.application_idx).
    */
   get applicationId(): number {
-    return +this.line.substr(13, 10);
+    return +this.line.substring(13, 13 + 10);
   }
   /**
    * Educational program start date (application_assessment.educ_period_start_dte).
    */
   get startDate(): Date | null {
-    return parseDate(this.line.substr(23, 8));
+    return parseDate(this.line.substring(23, 23 + 8));
   }
   /**
    * Educational Program End date (application_assessment.educ_period_end_dte).
    */
   get endDate(): Date | null {
-    return parseDate(this.line.substr(31, 8));
+    return parseDate(this.line.substring(31, 31 + 8));
   }
   /**
    * Program year (Application.program_yr_id).
    * Example: 20202021.
    */
   get programYearId(): number | null {
-    return +this.line.substr(39, 8) || null;
+    return +this.line.substring(39, 39 + 8) || null;
   }
   /**
    * Total BC Student Loan (award_disbursement.disbursement_amt).
    */
   get bslAward(): number {
-    return parseDecimal(this.line.substr(47, 10));
+    return this.line.substring(47, 47 + 10).trim()
```

Comment:
Can you please adjust the return type to reflect the possible null being returned now?
```
get bslAward(): number | null
```